README.md

```diff
@@ -1,25 +1,24 @@
 [![Build Status](https://travis-ci.org/iqbal-lab-org/cluster_vcf_records.svg?branch=master)](https://travis-ci.org/iqbal-lab-org/cluster_vcf_records)
 
 # cluster_vcf_records
-Python3 package to cluster VCF records. Used by gramtools and minos - written
-specifically for those projects and no others.
+Python3 package to cluster overlapping VCF records.
```

Comment:
I would leave this as before because the records do not have to overlap (minos merges non-overlapping records).